src/templates/daily_report/html.handlebars

```diff
@@ -0,0 +1,517 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
This is used for both daily and weekly reports and it also its coffeescript, so maybe move out of src/ and rename it to `templates/report.handlebars`